Mathlib/MeasureTheory/VectorMeasure/WithDensity.lean

```diff
@@ -67,7 +67,7 @@ theorem withDensityᵥ_neg : μ.withDensityᵥ (-f) = -μ.withDensityᵥ f := by
   · ext1 i hi
     rw [VectorMeasure.neg_apply, withDensityᵥ_apply hf hi, ← integral_neg,
       withDensityᵥ_apply hf.neg hi]
-    rfl
+    simp only [Pi.neg_apply]
```

Comment:
The `simp only` can't be combined with `rw`?

Reply:
No, it's rewriting under the integral.